engine/cld/config/network/config.go

```diff
@@ -187,29 +202,40 @@ func Load(filePaths []string, opts ...LoadOption) (*Config, error) {
 		cfg.Merge(&fileCfg)
 	}
 
-	// Apply the URL transformer if one is provided.
-	if loadCfg.RPCURLTransformer != nil {
-		cfg.transformRPCURLs(loadCfg.RPCURLTransformer)
+	// Apply the URL transformers if provided
+	if loadCfg.HTTPURLTransformer != nil {
+		cfg.transformHTTPURLs(loadCfg.HTTPURLTransformer)
+	}
+
+	if loadCfg.WSURLTransformer != nil {
+		cfg.transformWSURLs(loadCfg.WSURLTransformer)
 	}
 
 	return cfg, nil
 }
 
-// LoadOption defines a function that modifies the load configuration.
-type LoadOption func(*loadConfig)
+// loadOption defines a function which modifies the load configuration.
+type loadOption func(*loadConfig)
```

Comment:
This should be public right? Else it may be hard for consumer to work with option
```suggestion
// LoadOption defines a function which modifies the load configuration.
type LoadOption func(*loadConfig)
```

Reply:
Thats a good point. I'll change it back.

Thanks